test/features/test_rift_estimation.cpp

```diff
@@ -117,26 +117,12 @@ TEST (PCL, RIFTEstimation)
 
   // Compare to independently verified values
   const RIFTDescriptor &rift = rift_output[220];
-  float correct_rift_feature_values[32];
 
-  unsigned major, minor, patch;
-  std::sscanf (FLANN_VERSION_, "%u.%u.%u", &major, &minor, &patch);
-  if (PCL_VERSION_CALC (major, minor, patch) > PCL_VERSION_CALC (1, 8, 4))
-  {
-    const float data[32] = {0.0052f, 0.0349f, 0.0647f, 0.0881f, 0.0042f, 0.0131f, 0.0346f, 0.0030f,
-                            0.0076f, 0.0218f, 0.0463f, 0.0030f, 0.0087f, 0.0288f, 0.0920f, 0.0472f,
-                            0.0211f, 0.0420f, 0.0726f, 0.0669f, 0.0090f, 0.0901f, 0.1274f, 0.2185f,
-                            0.0147f, 0.1222f, 0.3568f, 0.4348f, 0.0149f, 0.0806f, 0.2787f, 0.6864f};
-    std::copy_n (data, 32, correct_rift_feature_values);
-  }
-  else
-  {
-    const float data[32] = {0.0187f, 0.0349f, 0.0647f, 0.0881f, 0.0042f, 0.0131f, 0.0346f, 0.0030f,
-                            0.0076f, 0.0218f, 0.0463f, 0.0030f, 0.0087f, 0.0288f, 0.0920f, 0.0472f,
-                            0.0076f, 0.0420f, 0.0726f, 0.0669f, 0.0090f, 0.0901f, 0.1274f, 0.2185f,
-                            0.0147f, 0.1222f, 0.3568f, 0.4348f, 0.0149f, 0.0806f, 0.2787f, 0.6864f};
-    std::copy_n (data, 32, correct_rift_feature_values);
-  }
+  const float correct_rift_feature_values[32] =
```

Comment:
```suggestion
    std::array<float, 32> correct_rift_feature_values{};
```
On this way you can use a for-ranges loop below.